ssn/chapters/Introduction.html

```diff
@@ -32,45 +36,63 @@ <h3>Context</h3>
   commands they expose to some network, and the physical systems upon which these devices act.
 </p>
 
-<h3>Purpose</h3>
+<h3>Scope</h3>
 
 <p>
-  This recommendation defines the Semantic Sensor Network (<b>SSN</b>) ontology.
-  It provides flexible but coherent perspectives for representing the entities, relations, and activities
-  involved in actuation, observation, and sampling.
+  This recommendation defines the Semantic Sensor Network Ontology (<b>SSN</b>).
+  SSN provides flexible but coherent perspectives for representing the entities, relations, and activities involved in
+  actuation, observation, and sampling, including the procedures executed, and the systems that implement the
+  procedures.
+  Supporting concepts include platforms, deployments, observed or actuated properties, features of interest, and
+  resulting samples, along with collections (sets) of executions and results that are typically involved in analysis and
+  implementation.
 </p>
+
 <p>
-  A lightweight core for SSN, known as <b>SOSA</b> (Sensor, Observation, Sample, and Actuator) aims to broaden the
+  A lightweight core module, known as <b>SOSA</b> (Sensor, Observation, Sample, and Actuator) aims to broaden the
```

Comment:
Different parts of speech make this a weird acronym. If it's not yet cast in stone, might I suggest a modified expansion?
```suggestion
  A lightweight core module, known as <b>SOSA</b> (Sense, Observe, Sample, and Actuate) aims to broaden the
```

Reply:
Alternatively...
```suggestion
  A lightweight core module, known as <b>SOSA</b> (Sensor, Observer, Sampler, and Actuator) aims to broaden the
```

Reply:
The terms `Sensor`, `Observation`, `Sample`, and `Actuator` are classes in the ontology. They also make up the name of the original specification (v1.0).

There should be room to review such things, but it would rather take shape as its own issue.